software/apps/romi_control/outputs.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * states.h
+ *
+ *  Created on: Sep 22, 2018
+ *      Author: shromonaghosh
+ */
+
+#ifndef OUTPUTS_H_
+#define OUTPUTS_H_
+
+#include <math.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+
+#include "app_error.h"
+#include "app_timer.h"
+#include "nrf.h"
+#include "nrf_delay.h"
+#include "nrf_gpio.h"
+#include "nrf_log.h"
+#include "nrf_log_ctrl.h"
+#include "nrf_log_default_backends.h"
+#include "nrf_pwr_mgmt.h"
+#include "nrf_drv_spi.h"
+
+#include "buckler.h"
+#include "display.h"
+#include "kobukiActuator.h"
+#include "kobukiSensorPoll.h"
+#include "kobukiSensorTypes.h"
+#include "kobukiUtilities.h"
+#include "lsm9ds1.h"
+#include "simple_ble.h"
+
+#include "helpers.h"
+
+typedef struct outputs {
+	bool notify_ack;
+	int notify_val;
```

Comment:
Is there a need for both a notify_ack and a notify_val? Couldn't we just check if notify_val has a non zero value?

Reply:
Yeah, sometimes we need to update the value of the ack without notifying the phone. This makes it easier.